apps/studio/docker-compose.yml

```diff
@@ -12,5 +12,19 @@ services:
     volumes:
       - postgres-volume:/var/lib/postgresql/data
 
+  mockpass:
+    build: https://github.com/isomerpages/mockpass.git
```

Comment:
issue: I know we forked because mockpass got some issues, but are we able to either:
1. Have it hosted on `opengovsg` org (we might deprecate `isomerpages` alr in the future, just making sure we don't forget)
2. Use a specific branch inside the original `mockpass` repo, then reference it from here?

Reply:
sure, i'll raise a PR back to mockpass for this